internal/vds/axis.cpp

```diff
@@ -0,0 +1,102 @@
+#include "axis.h"
```

Comment:
Looking at this I regret that we didn't go for `.hpp` for out c++ code. It would make a good destination between CGO and C++ headers. If you agree I suggest to  add a `git mv boundingbox.h boundingbox.hpp` commit to this PR and fixup this commit with a git mv too